src/redis.go

```diff
@@ -65,6 +66,16 @@ func main() {
 	}
 	defer conn.Close()
 
+	// Support using renamed commands
+	if args.RenamedCommands.Get() != nil {
+		renamedCommands, err := getRenamedCommands(args.RenamedCommands)
+		if err != nil {
+			log.Error("Renamed commands argument is invalid: %v", err)
+		} else {
+			conn.RenameCommands(renamedCommands)
+		}
```

Comment:
We don't want the integration to run with invalid args, I would exit with an error (e.g. with the elsewhere used `fatalIfErr`) if `getRenamedCommands` fails.